blocks/article-header/article-header.test.js

```diff
@@ -0,0 +1,11 @@
+describe('Article Header Block', () => {
+  beforeAll(async () => {
+    await page.goto(`${global.BASE_URL}ideas/example-article`);
+  });
+
+  it('should render the article header element', async () => {
+    await page.waitForSelector('.article-header');
+    const header = await page.$('.article-header');
```

Comment:
```suggestion
    const header = await page.waitForSelector('.article-header');
```
`waitForSelector` already returns the element, so we don't need to do it twice. I noticed that this is something we can update in some our tests.